src/components/patterns/main-map-wrapper/MainMapWrapper.vue

```diff
@@ -370,6 +395,7 @@ export default {
          * Updates active pins for map
          */
         filterPlaces(id) {
+            console.log('filter places');
```

Comment:
Can this be removed?

Reply:
Will merge this and remove in a future branch